docs/excel/excel-add-ins-multiple-ranges.md

```diff
@@ -0,0 +1,225 @@
+- `RangeAreas.copyFrom` can take either a `Range` or a `RangeAreas` parameter representing the source range(s) of the copy operation.
+
+### RangeArea-specific properties and methods
+
+The `RangeAreas` type has some properties and methods that are not on the `Range` object:
+
+- `areas`: A `RangeCollection` object that contains all of the ranges represented by the `RangeAreas` object. The `RangeCollection` object is also new and is similar to other Excel collection objects. It has an `items` property which is an array of `Range` objects representing the ranges.
+- `areaCount`: The total number of ranges in the `RangeAreas`.
+- `getOffsetRangeAreas`: Works just like [Range.getOffsetRange](https://docs.microsoft.com/javascript/api/excel/excel.range#getoffsetrange-rowoffset--columnoffset-), except that a `RangeAreas` is returned and it contains ranges that are each offset from one of the ranges in the original `RangeAreas`.
+
+## Create RangeAreas and set properties
+
+You can create `RangeAreas` object in two basic ways:
+
+- Call `Worksheet.getRanges()` and pass it a string with comma-delimited range addresses. If any range you want to include has been made into a [NamedItem](https://docs.microsoft.com/javascript/api/excel/excel.nameditem), you can include the name, instead of the address, in the string.
+- Call `Workbook.getSelectedRanges()`. This method returns a `RangeAreas` representing all the ranges that are selected on the currently active worksheet.
+
+Once you have a `RangeAreas` object, you can create others using the methods on the object that return `RangeAreas` such as `getOffsetRangeAreas` and `getIntersection`.
+
+> [!NOTE]
+> You cannot directly add additional ranges to a `RangeAreas` object. For example, the collection in `RangeAreas.areas` does not have an `add` method.
+
+
+> [!WARNING] 
+> Do not attempt to directly add or delete members of the the `RangeAreas.areas.items` array. This will lead to undesirable behavior in your code. For example, it is possible to push an additional `Range` object onto the array, but doing so will cause errors because `RangeAreas` properties and methods behave as if the new item isn't there. For example, the `areaCount` property does not include ranges pushed in this way, and the `RangeAreas.getItemAt(index)` throws an error if `index` is larger than `areasCount-1`. Similarly, deleting a `Range` object in the `RangeAreas.areas.items` array by getting a reference to it and calling its `Range.delete` method causes bugs: although the `Range` object *is* deleted, the properties and methods of the parent `RangeAreas` object behave, or try to, as if it is still in existence. For example, if your code calls `RangeAreas.calculate`, Office will try to calculate the range, but will error because the range object is gone.
```

Comment:
Isn't this the case with any of our collections?  (the part about pushing or removing items from the `.items` array, that is)

Reply:
@Zlatkovsky I was just wondering that myself driving home last night. But people will try this, so we need to warn them somewhere. We just need to put the warning somewhere more general and then link to it in this article.

Reply:
@Rick-Kirkham @Zlatkovsky I know there's a work item in our backlog to write a "Collections Guidance" article. Let's bump the priority of that up for the coming quarter and be sure to reference this article to link.

Reply:
@AlexJerabek @Zlatkovsky That sounds good to me. What's the bug number for that item?

Reply:
@Rick-Kirkham Here's the [internal link](https://office.visualstudio.com/OC/_workitems/edit/1828171?src=alerts&src-action=cta)

Reply:
@AlexJerabek I've added a note to that item about this discussion.